product-service/tests/healthcheck.test.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+import { API_URL } from '../src/constants'
+
+test('healthcheck returns 200 OK', async () => {
+    await axios.get(`${API_URL}/healthcheck`).then((response) => {
+        expect(response.status).toBe(200)
```

Comment:
Maksim, unit tests should not make real API calls. When you call real API calls it looks more like integration testing.
Your test cases should call lambda function directly, pass mock event to it and check results ;)